static/css/cf-enhancements.less

```diff
@@ -660,3 +660,34 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Full Width Buttons
+  family: cf-enhancements
+  notes:
+    - "Modifier to buttons full width at small screen sizes"
+  patterns:
+    - name: Default example
+      markup: |
+        <btn class="btn btn__full">Button Text</btn>
+  tags:
+    - cf-buttons
```

Comment:
The latter. For the FJ docs it should go into cf-enhancements family, but it should also eventually go into cf-buttons. Figured tagging it's future home would help guide whoever eventually moves it after testing.